src/Store/index.js

```diff
@@ -1,52 +1,13 @@
-import { applyMiddleware, combineReducers, createStore, compose } from 'redux';
 import promiseMiddleware from 'redux-promise-middleware';
 import { AdvisorStore } from '../AppReducer';
-
+import { getRegistry } from '@red-hat-insights/insights-frontend-components';
 let registry;
 
-class ReducerRegistry {
```

Comment:
Hard nope on removing this. Redux dev tools does NOT work when pulling in reducer registry.  Till it does, we gotta run with our own.

Reply:
Oh awesome! All the same, I would much prefer we unwrap that additional change from this pr, do it standalone if indeed it does fix the issue.... 🙇‍♀️

Reply:
Sure thing, good thinking.